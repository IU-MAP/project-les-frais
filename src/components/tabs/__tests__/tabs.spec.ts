import { mount } from '@vue/test-utils';
import Tabs from '../index.vue';

describe('Tabs', () => {
  it('matches the shapshot', () => {
    const component = mount(Tabs, { props: { items: ['item1', 'item2'], initial: 'item1' } });
    expect(component).toMatchSnapshot();
  });
});
