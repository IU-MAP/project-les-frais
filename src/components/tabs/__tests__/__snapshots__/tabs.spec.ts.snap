// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs matches the shapshot 1`] = `
VueWrapper {
  "__app": Object {
    "_component": Object {
      "__emits": null,
      "__props": Array [
        Object {},
        Array [],
      ],
      "name": "VTU_ROOT",
      "render": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      <div
        class="card tabs-nav"
      >
        
        <button
          class="active tab"
          type="button"
        >
          
          item1
          
        </button>
        <button
          class="tab"
          type="button"
        >
          
          item2
          
        </button>
        
      </div>
    </div>,
    "_context": Object {
      "app": [Circular],
      "components": Object {
        "transition": Object {
          "name": "transition",
          "props": undefined,
          "render": [Function],
        },
        "transition-group": Object {
          "name": "transition-group",
          "props": undefined,
          "render": [Function],
        },
      },
      "config": Object {
        "errorHandler": undefined,
        "globalProperties": Object {},
        "isCustomElement": [Function],
        "isNativeTag": [Function],
        "optionMergeStrategies": Object {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": Object {},
      "mixins": Array [
        Object {
          "__emits": null,
          "__props": Array [],
          "beforeCreate": [Function],
        },
      ],
      "provides": Object {},
      "reload": [Function],
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": Object {
      "errorHandler": undefined,
      "globalProperties": Object {},
      "isCustomElement": [Function],
      "isNativeTag": [Function],
      "optionMergeStrategies": Object {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.0.11",
  },
  "__setProps": [Function],
  "componentVM": Object {
    "active": "item1",
    "changeActive": [Function],
    "hasOwnProperty": [Function],
    "initial": "item1",
    "items": Array [
      "item1",
      "item2",
    ],
  },
  "rootVM": Object {},
  "wrapperElement": <div
    class="card tabs-nav"
  >
    
    <button
      class="active tab"
      type="button"
    >
      
      item1
      
    </button>
    <button
      class="tab"
      type="button"
    >
      
      item2
      
    </button>
    
  </div>,
}
`;
